src/module/treasure.js

```diff
@@ -1,32 +1,41 @@
 export const augmentTable = (table, html, data) => {
   // Treasure Toggle
+  const isTreasureTable = table.object.getFlag("ose", "treasure");
```

Comment:
getFlag() returns undefined for a table where the flag has not yet been created by the click event

Reply:
Good point, I didn't think of that either (still need to get used to the joy of `undefined` coming from C++). Would `getFlag() || false` enforce a boolean value?
This should be the only place where we change this flag for this Table object, but I don't know what would happen if something else put a flag that wasn't a boolean value on this object with this name.

Reply:
I just thought of it, but is it an accepted practice if I use `!!getFlag()` to force a boolean value?

Reply:
I think you can Boolean(getFlag()) too. I don't mind either way